CHANGES.md

```diff
@@ -5,6 +5,254 @@ all changes see the [closed
+CYLC_DIR is used in Cylc.
+
+[#3165](https://github.com/cylc/cylc-flow/pull/3165) - added GitHub
+Issue and Pull Request templates.
+
+### Fixes
+
+[#3010](https://github.com/cylc/cylc-flow/pull/3010) - fixes except KeyError
+in task_job_mgr.
+
+[#3031](https://github.com/cylc/cylc-flow/pull/3031) - convert range to list
+so that we can use reduce.
+
+[#3040](https://github.com/cylc/cylc-flow/pull/3040) - add check for zero
+to xrandom.
+
+[#3018](https://github.com/cylc/cylc-flow/pull/3018) - subprocpool: use
+SpooledTemporaryFile instead of TemporaryFile.
+
+[#3032](https://github.com/cylc/cylc-flow/pull/3032) - fix syntax for
+array access in cylc check-software.
+
+[#3035](https://github.com/cylc/cylc-flow/pull/3035) - fix scheduler#shutdown
+when reference-log option is used.
+
+[#3056](https://github.com/cylc/cylc-flow/pull/3056) - fix suite freeze on
+non-existent xtrigger.
+
+[#3060](https://github.com/cylc/cylc-flow/pull/3060) - fix bug when an
+AttributeError is raised, and add unit tests for xtrigger.
+
+[#3015](https://github.com/cylc/cylc-flow/pull/3015) - use global client
+zmq context.
+
+[#3092](https://github.com/cylc/cylc-flow/pull/3092) - fix recent
+shellcheck-inspired quoting errors.
+
+[#3085](https://github.com/cylc/cylc-flow/pull/3085) - fix cylc-search
+for directories without suite.rc.
+
+[#3105](https://github.com/cylc/cylc-flow/pull/3105) - fix work location
+assumption in a test.
+
+[#3112](https://github.com/cylc/cylc-flow/pull/3112) - prepend custom
+Jinja2 paths.
+
+[#3130](https://github.com/cylc/cylc-flow/pull/3130) - remove Python 2
+compatibility in setup.py for wheel.
+
+[#3137](https://github.com/cylc/cylc-flow/pull/3137) - fix job kill
+hold-retry logic.
+
+[#3077](https://github.com/cylc/cylc-flow/pull/3077) - support single
+port configuration for zmq.
+
+[#3153](https://github.com/cylc/cylc-flow/pull/3153) - fix bug in
+async_map.
+
+[#3164](https://github.com/cylc/cylc-flow/pull/3164) - fix pclient
+undefined error.
+
+### Documentation
+
+[#3025](https://github.com/cylc/cylc-flow/pull/3025) - fix dev-suites
+reference in docs (now in examples).
+
+[#3004](https://github.com/cylc/cylc-flow/pull/3004) - document suite
+runtime interface.
+
+[#3066](https://github.com/cylc/cylc-flow/pull/3066) - minor fix to
+docs about exit-script.
+
+[#3108](https://github.com/cylc/cylc-flow/pull/3108) - anatomy of a
+job script.
+
+[#3129](https://github.com/cylc/cylc-flow/pull/3129) - added
+SECURITY.md.
+
+[#3151](https://github.com/cylc/cylc-flow/pull/3151) - fix documentation
+heading levels.
+
+[#3158](https://github.com/cylc/cylc-flow/pull/3158) - fix \ in doco
+when wrapped in ``..``.
+
+### Security issues
+
+None. Note that we added a `SECURITY.md` file in this release (see #3129)
+with instructions for reporting security issues, as well as a
+listing with current incident reports.
+
+-------------------------------------------------------------------------------
+## __cylc-8.0a0 (2019-03-12)__
```

Comment:
Not a real release. Just a placeholder?

Reply:
I was no sure about this one... it was uploaded to PYPI, and we have a tag. The official 8.0.0 and 8.1.0 etc will be uploaded to PYPI and will have a tag too. So I was not sure whether this is an alpha *release*, or if we were not going to consider this a *release* yet?

Reply:
As I recall this was just a placeholder (to mark where we first branched off, or for PYPI?). 

We should probably group all cylc-8 changes under the first official (non-alpha) 8.0.0 release so that users are confronted with them all at once.  But I don't think it matters at this stage, we can keep the alpha heading to help order the change log comments for ourselves, for the moment.

Reply:
>we can keep the alpha heading to help order the change log comments for ourselves, for the moment.

All good for me to group everything under 8.0.0. And it should be a 2 min job to undo this change later too.

Reply:
(Does that mean you are going to change this now?)

Reply:
Was not planning to change that now. But can do if you prefer.

Reply:
No, that's fine - leave it.